Core/Goals/LootGoal.cs

```diff
@@ -123,6 +123,12 @@ public override async Task PerformAction()
                     return;
                 }
                 */
+                if (!await Wait(4000, DiDEnteredCombat()))
+                {
+                    await AquireTarget();
+                    return;
+                }
+
```

Comment:
why would you want to wait 4 seconds to find a target here?

Reply:
yes, It should be removed